packages/typespec-test/test/loadtesting_modular/tspconfig.yaml

```diff
@@ -12,6 +12,7 @@ options:
     enableOperationGroup: true
     isModularLibrary: true
     hierarchyClient: false
+    experimentalExtensibleEnums: true
```

Comment:
The extensible union in record has type mismatch issue which is not resolved yet.

I tentatively enable this option to by pass this issue.

Reply:
it should generate as any if it's not enabled, did you see any errors there?

Reply:
yeah, the case is Record of extensible enum. The modular is built as A | B but the RLC is built as string. So directly assigning would cause compile error.